utils/cudy-bdinfo/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2023 David Bauer <mail@david-bauer.net>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cudy-bdinfo
+PKG_SOURCE_DATE:=2023-06-01
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/blocktrron/cudy-bdinfo-decrypt.git
+PKG_SOURCE_VERSION:=5a60308c0fff610c4698207387d7153aba5fd62b
+PKG_MIRROR_HASH:=aefcd37e4b059d92226d2bc97b1b199f6a360c4935a6e92c930998b510275838
+
+PKG_MAINTAINER:=David Bauer <mail@david-bauer.net>
+PKG_LICENSE:=GPL-2.0-or-later
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/cudy-bdinfo
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=+libopenssl
```

Comment:
By default, this is going to be available for all routers supported within OpenWrt. I dont think this is desired behavior. You should add here also depends on the specific target or even subtarget for Cudy routers.

Reply:
Interesting that openssh is used instead of mbedtls. The latter is default with OpenWrt.

Reply:
@neheb I was not able to decrypt the content using mbedtls. I won't rule out this is possible though.